.gitignore

```
@@ -1 +1,3 @@
-__pycache__
+__pycache__/
+
+datasets/
```

but why ignore our datasets ? 🥺
```suggestion
datasets/
```

---

the datasets should been saved in the local memory disk for user and should not be tracked by github. This is similar to `player_data.json` in `zdd_adventure`.

If we track the datasets, I worry we would have a merge conflict later due to inconsistency of the datasets.

What do you think about it?